transform/mattermost-analytics/dbt/sources.yml

```diff
@@ -0,0 +1,845 @@
+[0m12:25:13  Running with dbt=1.4.1
+[0m12:25:14  Unable to do partial parsing because config vars, config profile, or config target have changed
+[0m12:25:21  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
+There are 3 unused configuration paths:
+- models.mattermost_analytics.marts
+- models.mattermost_analytics.utilities
+- models.mattermost_analytics.intermediate
+[0m12:25:26  version: 2
```

Comment:
should we remove this paragraph?

Reply:
Yes, you are right. I'll actually also add it in `.gitignore`.